components/PodHeader.vue

```diff
@@ -1,7 +1,8 @@
-<script setup>
+<script setup lang="ts">
 	import { ChevronDownIcon } from '@heroicons/vue/24/outline'
-	const { me } = useApi()
-	const customer = await me()
+	import type { Customer } from '~/types'
+	const { getMe } = useCustomerStore()
+	const customer: Customer = await getMe()
```

Comment:
Same thing here, we don't need to call the api again because it's already called in `startApp` and the state is stored in the store, allowing us to use it everywhere in the app.